gulpfile.js

```diff
@@ -93,22 +93,6 @@ gulp.task('sass:build', ['webfont'], function() {
     }))
```

Comment:
Take care, if I'm not wrong you're undoing the changes that @cbernat did.

Reply:
Yes @IgnacioRodrigues  please do not remove this, else dist task will cease to work again.

Reply:
Are you sure this works correctly? I deleted it because it was not working properly for me.

Reply:
It's okey, we already reviewed it with @cbernat